### ISSUE TITLE
The pruned model is as the same as the original one

### ISSUE THREAD

**Issue by dreamlychina** [microsoft/nni#5277]
**Describe the issue**:
The pruned model is as the same as the original one。 What does “module.up2.conv.double_conv.2 has mask, but not found in the traced graph, just skip it.” mean？ How can I fix it？ The original op name is  “up2.conv.double_conv.2”， but it changed to "module.up2.conv.double_conv.2".

The log is:
''''''''''''''''''''''''''''''''''''''''''
[12/12/2022, 11:34:07 AM] INFO (nni.compression.torch.utils.mask_conflict) {'inc.double_conv.0': 1, 'inc.double_conv.2': 1, 'down1.pool_conv.1.double_conv.0': 1, 'down1.pool_conv.1.double_conv.2': 1, 'down2.pool_conv.1.double_conv.0': 1, 'down2.pool_conv.1.double_conv.2': 1, 'down3.pool_conv.1.double_conv.0': 1, 'down3.pool_conv.1.double_conv.2': 1, 'down4.pool_conv.1.double_conv.0': 1, 'down4.pool_conv.1.double_conv.2': 1, 'up1.conv.double_conv.0': 1, 'up1.conv.double_conv.2': 1, 'up2.conv.double_conv.0': 1, 'up2.conv.double_conv.2': 1, 'up3.conv.double_conv.0': 1, 'up3.conv.double_conv.2': 1, 'up4.conv.double_conv.0': 1, 'up4.conv.double_conv.2': 1, 'outc.conv': 1}
[12/12/2022, 11:34:07 AM] WARNING (nni.compression.torch.utils.mask_conflict) no multi-dimension masks found.

[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up2.conv.double_conv.2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up2.conv.conv1 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up2.conv.conv2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.double_conv_batch_norm.0 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.double_conv_batch_norm.3 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.double_conv.0 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.double_conv.2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.conv1 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up3.conv.conv2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.double_conv_batch_norm.0 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.double_conv_batch_norm.3 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.double_conv.0 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.double_conv.2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.conv1 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.up4.conv.conv2 has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] WARNING (nni.compression.torch.speedup.compressor) module.outc.conv has mask, but not found in the traced graph, just skip it.
[12/12/2022, 11:23:13 AM] INFO (nni.compression.torch.speedup.compressor) replace compressed modules...
[12/12/2022, 11:23:13 AM] INFO (nni.compression.torch.speedup.compressor) speedup done
elapsed time when use speedup:  0.0017943382263183594
the outputs from use_mask and use_speedup are the same
’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’‘’

**Environment**: 
- NNI version: 1.9
- Training service (local|remote|pai|aml|etc): remote
- Client OS:
- Server OS (for remote mode only):
- Python version: 3.8.3
- PyTorch/TensorFlow version: 1.4.0
- Is conda/virtualenv/venv used?: virtualenv
- Is running in Docker?: Yes

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by Lijiaoa** [comment]
Hi, @dreamlychina Thanks for raising this issue! Could you upgrade your NNI version to v2.10 and run the model again?

**Comment by J-shang** [comment]
Closed due to no reply for a long time, please reopen if have any questions.
